pkg/trace/api/profiles.go

```diff
@@ -68,7 +74,20 @@ func (r *HTTPReceiver) profileProxyHandler() http.Handler {
 	if err != nil {
 		return errorHandler(err)
 	}
-	tags := fmt.Sprintf("host:%s,default_env:%s", r.conf.Hostname, r.conf.DefaultEnv)
+	ver, err := version.Agent()
+	verStr := "unknown"
+	if err == nil {
+		verStr = ver.GetNumberAndPre()
+	}
+
+	tags := fmt.Sprintf("host:%s,default_env:%s,agent_version:%s", r.conf.Hostname, r.conf.DefaultEnv, verStr)
```

Comment:
```suggestion
	tags := fmt.Sprintf("host:%s,default_env:%s,agent_version:%s", r.conf.Hostname, r.conf.DefaultEnv, info.Version)
```

`info.Version` is what you need and what we use everywhere.

Reply:
info.Version returns "" when i run the tests

Reply:
Doesn't matter, simply set it to whatever you like for the sake of the test if you will. Like I said, it's what we use everywhere for tagging, and it is set in every Datadog Agent package at build time. It's guaranteed to be set, and correct.

Reply:
setting `info.Version` to whatever you like at the start of a test won't cause race conditions with other tests? Seems unsound.

Reply:
@peterg17 see https://github.com/DataDog/datadog-agent/blob/93673c4c5ce3795beaf6b604437df20edb962b79/pkg/trace/api/info_test.go#L93

This is not a great pattern and will create race conditions if tests in different packages are run in parallel, but that's the established pattern here.

Reply:
I'm doing this now, also i'm not checking for a specific version anyways just that the tag exists